### PR TITLE
remove unnecessary encoder

### PR DESCRIPTION
**Pull request by deads2k**
Followups to https://github.com/kubernetes/kubernetes/pull/63105

### CODE REVIEW

File: pkg/kubectl/cmd/get/humanreadable_flags.go [hunk 1]
```diff
@@ -66,7 +66,6 @@ func (f *HumanPrintFlags) ToPrinter(outputFormat string) (printers.ResourcePrint
 		return nil, printers.NoCompatiblePrinterError{Options: f}
 	}
 
-	encoder := scheme.Codecs.LegacyCodec(scheme.Registry.RegisteredGroupVersions()...)
```
Comment (deads2k) [inline pkg/kubectl/cmd/get/humanreadable_flags.go]: @liggitt one does has to wonder what exactly you'd be re-encoding.  Might be able to remove the entire raw case.

**Comment by deads2k** [comment]
/assign @juanvallejo 

@kubernetes/sig-cli-misc

**Review by juanvallejo — APPROVED** [review]

**Pull request merged**